### DISCUSSION TITLE
Password Validator

### DISCUSSION THREAD

**Discussion by ValliammaiMuthiah** [Azure/api-management-developer-portal#2597] [Feature request]
**Every bug report should have precise description and reproduction steps; console traces or source code references are appreciated.**

**For assistance requests, contact Azure support or submit a post [on Stack Overflow](http://aka.ms/apimso). We don't provide support through GitHub Issues. Feature requests can be raised [on the Azure Feedback Forum](https://aka.ms/apimwish).**

## Bug description

Sequence of alphabets / numbers are accepted by password parameter.

## Reproduction steps

1. Go to Sign Up page.
2. Fill all the inputs.
3. In password & confirm password, give sequence of characters / numbers.
4. Click the sign up button.
5. It'll accept the sequence of password without validation message.

## Expected behavior

It should throw validation message when sequence is given in password.
Similar to deprecated portal, 4 stages of validation[eg: weak, medium] would be helpful to make passwords stronger.

## Is your portal managed or self-hosted?

Self-hosted

## Release tag or commit SHA (if using self-hosted version)

R 18

## Environment

 - Operating system: [e.g., iOS]
 - Browser: [e.g., Google Chrome, Safari]
 - Version: [e.g., 22]

## Additional context
![password validator](https://user-images.githubusercontent.com/75632710/182821145-5fa546b8-019e-46ec-b9e5-dbf0e77598e0.PNG)

**Comment by ghost** [comment]
@ValliammaiMuthiah, thank you for opening this issue. We will triage it within the next few business days.

**Comment by ghost** [comment]
@ValliammaiMuthiah, thank you for requesting this feature.
